tasks/slurmctld.yml

```diff
@@ -25,6 +25,17 @@
     state: directory
   when: slurm_create_dirs and __slurm_config_merged.SlurmctldLogFile != omit
 
+- name: Add slurmctld service
+  ansible.builtin.copy:
+    src: slurmctld.service
+    dest: /etc/systemd/system/slurmctld.service
+    owner: root
+    group: root
+    mode: 0644
+  when: slurm_configure_repos
```

Comment:
I wonder if we should add a new variable `slurm_configure_systemd` (default `false`) to setup systemd unit files.

It is legit to use a custom repo with packages that include the systemd unit files. Thus, there should be no correlation between using a custom repo and having the systemd unit files installed by the role.

Reply:
i added this new variable ( slurm_configure_systemd ) to connection systemd unit files and this condition